docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -201,23 +246,15 @@ PUT _cluster/settings
 [[remote-cluster-settings]]
 === Remote cluster settings
 
-`cluster.remote.connections_per_cluster`::
-
-  The number of gateway nodes to connect to per remote cluster. The default is
-  `3`.
+`cluster.remote.${cluster_alias}.mode`::
```

Comment:
Perhaps add a sentence above the first setting stating that this section is covering settings that apply to both sniff and proxy mode, and that mode-specific settings are coming later.